QuestPaths/5.x - Shadowbringers/Side Quests/Il Mheg/3408_Scents of Security.json

```diff
@@ -0,0 +1,131 @@
+{
+  "$schema": "https://qstxiv.github.io/schema/quest-v1.json",
+  "Author": "alydev",
+  "QuestSequence": [
+    {
+      "Sequence": 0,
+      "Steps": [
+        {
+          "DataId": 1027670,
+          "Position": {
+            "X": 32.028687,
+            "Y": 101.900696,
+            "Z": -857.87744
+          },
+          "TerritoryId": 816,
+          "InteractionType": "AcceptQuest",
+          "Fly": true,
+          "AetheryteShortcut": "Il Mheg - Pla Enni",
+          "SkipConditions": {
+            "AetheryteShortcutIf": {
+              "NearPosition": {
+                "Position": {
+                  "X": 32.028687,
+                  "Y": 101.900696,
+                  "Z": -857.87744
+                },
+                "TerritoryId": 816,
+                "MaximumDistance": 200
+              }
+            }
+          }
+        }
+      ]
+    },
+    {
+      "Sequence": 1,
+      "Steps": [
+        {
+          "Position": {
+            "X": 32.028687,
+            "Y": 101.900696,
+            "Z": -857.87744
+          },
+          "TerritoryId": 816,
+          "InteractionType": "UseItem",
+          "ItemId": 2002608,
+          "$": "Starts with 0 16 0 0 0 0, using item sets to 0 0 0 0 0 0",
+          "CompletionQuestVariablesFlags": [
```

Comment:
Per my liveblogging in discord: the schema defines a Negative value for these objects, but it's not implemented. Mode:Exact seems to do the same time